code/src/reducers/directions.js

```diff
@@ -0,0 +1,67 @@
+import { createSlice } from '@reduxjs/toolkit'
+
+import { API_START, API_MOVE } from '../reusables/urls'
+
+export const directions = createSlice({
+  name: "directions",
+  initialState: {
+    name: '',
+    player: '',
+    step: localStorage.getItem('directions') 
+      ? JSON.parse(localStorage.getItem('directions'))
+      : null, 
+    history: null, 
+    loading: false
+  },
+  reducers: {
+    setPlayer: (store, action) => {
+      const uniqid = require('uniqid')
+      store.name = action.payload
```

Comment:
This was a nice and clever idea to implement a functionality like this! It's very user friendly! 👍